app/src/main/java/com/conupods/OutdoorMaps/BuildingOverlays.java

```diff
@@ -22,22 +22,38 @@
 public class BuildingOverlays {
 
     private final static String mBuildingLogTag = "GeoJsonOverlay";
-    private final static String mGeoStringLink = "https://gist.githubusercontent.com/carlitalabbe/87585a7c681a72c9a335cd282acee35d/raw/1740cf0f0d8bea67adbb133db7c60956d72c130c/map.geojson";
```

Comment:
is this being called from somewhere else now?

Reply:
nvm i seeing its an input too BuildingOverlay but isnt this something that is final?

Reply:
this class does buildingOverlays, maybe later we will overlay something different. it is to make it more modular/reusable

Reply:
sounds good